jib-core/build.gradle

```diff
@@ -13,8 +13,8 @@ plugins {
 
 group 'com.google.cloud.tools'
 
-sourceCompatibility = JavaVersion.VERSION_1_8
-targetCompatibility = JavaVersion.VERSION_1_8
+sourceCompatibility = JavaVersion.VERSION_11
+targetCompatibility = JavaVersion.VERSION_11
```

Comment:
Um… won't this result in using Java 11 bytecodes and .class file markers, preventing its use on Java 8?

Reply:
Yeah, this is for reference only and be merged probably a few years later.